src/material-experimental/mdc-radio/_radio-theme.scss

```diff
@@ -19,6 +19,8 @@
     $mdc-radio-unchecked-color: rgba(mdc-theme-prop-value(on-surface), 0.54) !global;
     $mdc-radio-disabled-circle-color: rgba(mdc-theme-prop-value(on-surface), 0.38) !global;
 
+    @include mdc-form-field-core-styles($query: mdc-helpers.$mat-theme-styles-query);
```

Comment:
Should this be nested under `mat-mdc-radio-button`? That's what we're doing for the checkbox.

Reply:
+1, this may end up affecting other components on the same page

Reply:
Cool yup - nested it under the host classname, and did the same for slide toggle